src/resources/favorites.ts

```diff
@@ -1,18 +1,24 @@
+import { UniqueId } from '@/__swaps__/types/assets';
+import { ChainId } from '@/__swaps__/types/chains';
 import { EthereumAddress, NativeCurrencyKeys, RainbowToken } from '@/entities';
 import { Network } from '@/networks/types';
 import { createQueryKey, queryClient } from '@/react-query';
 import { DAI_ADDRESS, ETH_ADDRESS, SOCKS_ADDRESS, WBTC_ADDRESS, WETH_ADDRESS } from '@/references';
+import { promiseUtils } from '@/utils';
 import ethereumUtils, { getUniqueId } from '@/utils/ethereumUtils';
```

Comment:
FYI this is using old uniqueId pattern: `<address>_<network>` instead of `<address>_<chainId>`. It's probably functioning fine, but if we're eventually going to migrate stuff to chainId, might be worth doing now?